bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -309,6 +309,27 @@ private synchronized void stopPanelDiscoveryJob() {
         }
     }
 
+    private synchronized void startTouchJob() {

```

Comment:
Would it make sense to only start the job if the deviceType is "canvas"?

Reply:
Yep, that's a good idea (if setting the type to canvas works which I will check asap so I can rely on that)